src/main/java/talkabot/TalkaBot.java

```diff
@@ -0,0 +1,166 @@
+package talkabot;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+
+import talkabot.exceptions.InvalidEditException;
+import talkabot.exceptions.InvalidScheduleException;
+import talkabot.exceptions.NoInputException;
+import talkabot.exceptions.TalkaBotException;
+import talkabot.exceptions.UnknownInputException;
+import talkabot.task.Deadline;
+import talkabot.task.Event;
+import talkabot.task.Task;
+import talkabot.task.TaskList;
+import talkabot.task.ToDo;
+
+/**
+ * TalkaBot Class handles the running of Talk-a-Bot
+ * and responds according to user input.
+ */
+public class TalkaBot {
+
+    private Ui ui;

```

Comment:
I wonder if it would be better to use the full form for ui instead of an acronym?